src/include/migraphx/quantization.hpp

```diff
@@ -49,6 +49,8 @@ MIGRAPHX_EXPORT void quantize_int8(program& prog,
 MIGRAPHX_EXPORT void
 quantize_fp8(program& prog, const target& t, const std::vector<parameter_map>& calibration);
 
+MIGRAPHX_EXPORT void quantize_int4(program& prog);
```

Comment:
This should be called `quantize_int4_weights` since it will only quantize the weights.